cmd/subnetcmd/deploy.go

```diff
@@ -678,3 +692,45 @@ func PrintDeployResults(chain string, subnetID ids.ID, blockchainID ids.ID, isFu
 	table.Render()
 	return nil
 }
+
+// Has side effects and will set the version of avalanchego in the package-level avagoVersion variable.
```

Comment:
can we avoid the side effects? not so easy to follow flow

Reply:
removed side effect